mne/channels/channels.py

```diff
@@ -440,18 +440,36 @@ def inst_has(attr):
 class InterpolationMixin(object):
     """Mixin class for Raw, Evoked, Epochs
     """
-    def interpolate_bads_eeg(self):
-        """Interpolate bad channels
 
-        Operates in place.
+    def interpolate_bads(self, reset_bads=True):
```

Comment:
>
> but isn't this what is done in most epoch methods?
>

if you think that you might want to chain methods return self.

but don't write:

evoked_interp = evoked.interpolate_bads()

if interpolate_bads has no copy parameter and operates in place.

Sorry I do what I can to review from my phone...